include/script.h

```diff
@@ -5,8 +5,22 @@
 #include <stdbool.h>
 #include "utils/list.h"
 
+#define SCRIPT_VERSION 1
+
+/* used only by external script files */
+#ifndef UFTRACE_SCRIPT_H
+#if __cplusplus
+extern "C"
+#endif
+int uftrace_script_version(void)
+{
+	return SCRIPT_VERSION;
+}
```

Comment:
A function definition in the header?  I'm afraid that a plugin can consist of multiple files.

Reply:
Hmm.. Otherwise, the script writer has to implement this. Could you recommend if you have a better idea?

Reply:
I think it's script writer's responsibility to check the version number.